core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -336,9 +338,9 @@ protected void tickCapture(Duration duration) {
     }
 
     if (lead > 0) {
-      this.dominateAndFireEvents(leader, calculateDominateTime(lead, duration));
+      this.dominateAndFireEvents(leader, calculateDominateTime(lead, duration), contested);
     } else {
-      this.dominateAndFireEvents(null, duration);
+      this.dominateAndFireEvents(null, duration, contested);
     }
```

Comment:
```java
    if (lead > 0) {
      this.dominateAndFireEvents(leader, calculateDominateTime(lead, duration), false);
    } else {
      this.dominateAndFireEvents(null, duration, leader != null);
    }
```
If there's a lead > 0, the point is *not* contested, so just use false.
If there's not a lead, that means the point is contested *if* anyone is on it (meaning, there'd be some leader team)